### ISSUE TITLE
Speed up image loading

### ISSUE THREAD

**Issue by oeway** [Kitware/itk-vtk-viewer#219]
I noticed that the loading time for the viewer is a bit long, and did some profiling for that. I loaded a png image (1000x1000, RGB) locally from my disk, and it took me ~4 seconds to see the image.  Then I noticed that most of time is to load [ImageIO.worker.js](https://unpkg.com/itk@10.2.0/WebWorkers/ImageIO.worker.js)(**1.08s**) and  [itkPNGImageIOJSBindingWasm.js](https://unpkg.com/itk@10.2.0/ImageIOs/itkPNGImageIOJSBindingWasm.js)(**2.5s**).

![Screen Shot 2020-01-16 at 5 03 17 PM](https://user-images.githubusercontent.com/478667/72542393-5ddfad80-3884-11ea-8f3f-79c8971639d7.png)

So now the question is, can we not provide some caching for these files (e.g. prefetch these files) in the PWA to improve the performance? Or provide options for the viewer to preload these js files (since we know exactly the type of file formats we want to support, and subset of features we want to use)?

**Comment by oeway** [comment]
When using the viewer alone, it appears that the service worker is not running. For example, I cannot see the service worker for itk-vtk-viewer if I load it in ImJoy: https://imjoy.io/lite?plugin=oeway/ImJoy-Plugins:itk-vtk-viewer


Also, it seems the service worker is not functioning on https://kitware.github.io/itk-vtk-viewer/app/

Here is the error:
```
serviceWorker.js:1 Uncaught DOMException: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'https://kitware.github.io/itk-vtk-viewer/app/workbox-sw.prod.v2.1.2.js' failed to load.
    at https://kitware.github.io/itk-vtk-viewer/app/serviceWorker.js:1:1
(anonymous) @ serviceWorker.js:1
A bad HTTP response code (404) was received when fetching the script.
```

**Comment by thewtex** [comment]
@oeway good catch, and thanks for reporting this!

The build system for Workbox/Webpack was brittle, and the Workbox script was not being imported... I've updated from Workbox 2.X to 4.X, tested locally (nice new output in Chrome console!), and this is fixed in #226
